InventorAccessPortal.Web/Models/Account/RegisterModel.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Web;
+using InventorAccessPortal.Web.Util;
+
+namespace InventorAccessPortal.Web.Models.Account
+{
+    public enum RegisterErrorCodes
+    {
+        [Description("Invalid Username or Password.")]
+        InvalidUsernameOrPassword = 0,
+        [Description("Username or Password is empty.")]
+        EmptyUsernameOrPassword = 1
+    }
+    public class RegisterModel : SharedModel
+    {
+        public string Username { get; set; }
+        public string Email { get; set; }
+        public string Password { get; set; }
+        public string PasswordVerify { get; set; }
```

Comment:
PasswordVerify shouldn't be sent to the backend. It shouldn't let you submit from the font if the two passwords don't match. Only send back the password if both passwords match